src/main/java/dk/kb/netarchivesuite/solrwayback/service/dto/ArcEntry.java

```diff
@@ -11,14 +13,31 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.sun.tools.javac.api.Formattable;
```

Comment:
This is not used and it is a Sun/Oracle-specific class, for it makes it not work under OpenJDK.

Reply:
Bummer! Huge mistake. It is not even used